scripts/main.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const randomNumber = generateRandomNumber();
+let attempts = 5;
+
+document.querySelector('.form').onsubmit = function() {
+  const enteredNumber = this.querySelector('input').value;
+  const result = bullsAndCows(randomNumber, enteredNumber);
+
+  result && attempts--;
+
+  if (attempts < 1 || result && result.bulls === 4) {
+    gameOver(result);
+  } else {
+    fillUpOutput(enteredNumber, result);
+  }
+
+  return false;
+};
+
+function gameOver(result) {
+  outputElement.querySelector('.output__list').remove('output--active');
+
+  const gameOverElement = outputElement.querySelector('.output__gameover');
+
+  gameOverElement.classList.add('output__gameover--active');
+
+  gameOverElement.innerHTML = result.bulls === 4
+    ? 'You won!' : 'You lose!';
+
+  const message = document.createElement('span');
+
+  message.innerHTML = `The number was ${randomNumber}`;
+  outputElement.append(message);
+}
+
+const outputElement = document.querySelector('.output');
+
+function fillUpOutput(enteredNumber, result) {
+  outputElement.querySelector('.output__entered').innerHTML = enteredNumber;
+
+  outputElement.querySelector('.output__result').innerHTML
+    = result ? `Bulls: ${result.bulls}, Cows: ${result.cows}` : 'Wrong input!';
```

Comment:
do not break line just before/after `=`, `=>`
```suggestion
  outputElement.querySelector('.output__result').innerHTML = result
    ? `Bulls: ${result.bulls}, Cows: ${result.cows}`
    : 'Wrong input!';
```